en/docs/install-and-setup/setup/deployment-best-practices/monitoring-transaction-counts.md

```diff
@@ -0,0 +1,138 @@
+# Monitoring Integration Transactions Counts
+
+A **Transaction** in WSO2 Micro Integrator is typically defined as an inbound request (a request coming to the server). That is, any inbound request to a [REST API]({{base_path}}/integrate/develop/creating-artifacts/creating-an-api), [Proxy service]({{base_path}}/integrate/develop/creating-artifacts/creating-a-proxy-service), or [Inbound Endpoint]({{base_path}}/integrate/develop/creating-artifacts/creating-an-inbound-endpoint) is considered as one transaction.
+
+However, when the Micro Integrator is configured as both the message producer and consumer to handle **asynchronous** messaging scenarios, the two requests (listening request and sending request) are considered as a single transaction.
+
+If you need to track the number of transactions in your Micro Integrator deployment, you can enable the transaction counter component in each Micro Integrator instance of your deployment. Currently, the transaction counter is responsible for counting all requests received via the [HTTP Passthru]({{base_path}}/install-and-setup/setup/mi-setup/transport_configurations/configuring-transports/#configuring-the-httphttps-transport) and [JMS]({{base_path}}/install-and-setup/setup/mi-setup/transport_configurations/configuring-transports/#configuring-the-jms-transport) transports and for persisting the summary of the transaction count in a database for future use.
+
+Follow the instructions given below.
+
+## Step 1 - Enabling the transaction counter
+
+Configure a relational database to persist transaction count information and then enable the **Transaction Counter** component from the `deployment.toml` file (stored in the `<MI_HOME>/conf` folder).
+
+1.  Select the preferred database type from the list given below and follow the relevant link to set up a database.
+
+    - [Setting up a MySQL database]({{base_path}}/install-and-setup/setup/mi-setup/databases/setting-up-MySQL)
```

Comment:
We don't have these links now in APIM. Let's point to the relevant MI doc instead. https://mi.docs.wso2.com/en/4.2.0/install-and-setup/setup/deployment-best-practices/monitoring-transaction-counts/